pom.xml

```diff
@@ -292,22 +287,54 @@
                 </configuration>
             </plugin>
 
-            <plugin>
-                <groupId>com.diffplug.spotless</groupId>
-                <artifactId>spotless-maven-plugin</artifactId>
```

Comment:
I removed spotless because I thought openrewrite could do it's job. We have checks for the imports but do we want to additionally fix the issues with spotless?

Reply:
Can we please keep the plugin and have it perhaps deactivated by default? I find it super-useful to remove all unused imports with `mvn spotless:apply`.